multiversx_sdk_core/transaction_factories/transaction_factory_config.py

```diff
@@ -1,34 +1,33 @@
 from multiversx_sdk_core.address import DEFAULT_HRP, Address
 from multiversx_sdk_core.constants import \
     DEFAULT_EXTRA_GAS_LIMIT_FOR_GUARDED_TRANSACTION
-from multiversx_sdk_core.interfaces import (IAddress, IChainID, IGasLimit,
-                                            ITransactionValue)
+from multiversx_sdk_core.interfaces import IAddress
 
 
 class TransactionFactoryConfig:
-    def __init__(self, chain_id: IChainID) -> None:
+    def __init__(self, chain_id: str) -> None:
```

Comment:
Maybe we should name them `transaction intent factories`? If so, we should also update the specs.